CodeStarterCamp_Week3/main.swift

```diff
@@ -8,5 +8,59 @@
 
 import Foundation
 
-print("Hello, World!")
+class Person {
+    var name: String
+    var gender: String
+    var moneyOnHand: Int
+    var job: String
+    init(name: String, gender: String, moneyOnHand: Int, job: String) {
+        self.name = name
+        self.gender = gender
+        self.moneyOnHand = moneyOnHand
+        self.job = job
+    }
+    func buyCoffee (orderedCoffee: Coffee) {
+        print("\(orderedCoffee.rawValue) 주세요.")
+    }
```

Comment:
```suggestion
    func buy(_ coffee: Coffee) {
        print("\(coffee.rawValue) 주세요.")
    }
```

이렇게 하면 네이밍에 중복되는 단어사용을 피할 수 있을 것 같아요 . 또한 함수명과 소괄호 사이에 공백은 이 부분에만 들어갔네요! 다른곳과 같도록 통일시켜 주는 것이 좋습니다

Reply:
앗 이부분 공백은 수정하였습니다. 그런데 중복되는 단어는 위와 같이 buy(_ coffee) 로 표현하는 건가요..? 네이밍 규칙중 하나인가요???

Reply:
아니요! 정해진 규칙은 아닙니다. 가독성 측면에서 반복되는 단어 사용이 보기 좋지 않기 때문에 제가 자주 사용하는 방법이에요.
제가 드리는 코멘트는 정답이니 이렇게 바꿔야 한다! 라고 말씀드리는 것이 아닌, 다른방향을 제안 드리는 것이고 졔 님이 판단 후, 졔 님의 스타일대로 코드에 적용해주셔도 좋을 것 같아요 ☺️
파라미터 네임에 `_` 는 함수 사용(호출) 시, 파라미터명을 생략 해주는 기능을 합니다. 이 부분도 알아보시면 좋을 것 같아요 🧐